x/ccv/democracy/staking/module.go

```diff
@@ -50,6 +50,9 @@ func NewAppModule(cdc codec.Codec, keeper keeper.Keeper, ak types.AccountKeeper,
 // however, it returns no validator updates as validators are tracked via the
 // consumer chain's x/cvv/consumer module and so this module is not responsible
 // for returning the initial validator set.
+//
```

Comment:
This democracy staking keeper is only for governance capabilities, where ccv module deals with POS/block production capabilities